components/text/diffutils/patches/02-Makefile.in.patch

```diff
@@ -0,0 +1,14 @@
+Without this patch gdiff3 and gsdiff don't
+find diff - program that both of them call.
+
+--- diffutils-3.8/src/Makefile.in	2021-08-02 03:20:12.000000000 +0000
++++ diffutils-3.8/src/Makefile.in	2021-08-03 09:20:26.738482217 +0000
+@@ -1613,7 +1613,7 @@ noinst_HEADERS = \
+   system.h
+ 
+ MOSTLYCLEANFILES = paths.h paths.ht
+-gdiff = `echo diff|sed '$(transform)'`
++gdiff = `echo /usr/gnu/bin/diff`
```

Comment:
You should not need the echo, just assign the variable.

Reply:
Did it, thanks.